src/scipp/plotting/displayable.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+
+from ipywidgets import Widget
```

Comment:
Now this is a mandatory dependency again...

Reply:
Shouldn't be? I think i tested it? Nothing gets imported in the init until you call `plot`.

Reply:
To clarify: `ipywidgets` is a hard dependency if you want to plot (which it already is on `main`), but it is not a hard dependency to import scipp.
Making `ipywidgets` a soft dependency for plots will be done in a later PR.

Reply:
Mandatory for plotting, but as discussed offline this will be fixed independently (not for this patch release).

Reply:
see #2805